train_model.py

```diff
@@ -1,14 +1,17 @@
+# pylint: disable=wrong-import-order
```

Comment:
why is that needed?

Reply:
It isn't :D I was having issues with my local ruff-formatter. Fixed in the latest commit.

Reply:
Aha, I just realized that pylint and isort do not agree on the order either:
`train_model.py:10:0: C0411: third party import "from lightning_fabric.utilities import seed" should be placed before "import wandb" (wrong-import-order)` 
If it is okay with you, I would ignore the message for now.

Reply:
I fixed this in a PR to this branch